tests/utils/test_validation.py

```diff
@@ -195,6 +196,15 @@ def test_validate_experiment_name():
             _validate_experiment_name(invalid_name)
 
 
+def test_validate_experiment_pagination():
```

Comment:
Prefer to not test internal functions, but focus on testing public APIs. For example, in this case, prefer to actually call mlflow.list_experiments() with out of range max_results as an argument and assert failure from there. I see it's already tested to some extent in test_list_experiments_paginated_errors().

Reply:
Makes sense. I'll replace `_validate_experiment_pagination` with `mlflow.list_experiments()`.